easybuild/easyconfigs/l/libcroco/libcroco-0.6.11-intel-2016a.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'ConfigureMake'
+
+name = 'libcroco'
+version = '0.6.11'
+
+homepage = 'https://github.com/GNOME/libcroco'
+description = """Libcroco is a standalone css2 parsing and manipulation library."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+sources = [SOURCE_TAR_XZ]
+source_urls = ['http://ftp.gnome.org/pub/GNOME/sources/libcroco/%(version_major_minor)s/']
+
+dependencies = [
+    ('zlib', '1.2.8'),
+    ('libxml2', '2.9.3'),
+    ('GLib', '2.48.0'),
+    ('XZ', '5.2.2',)
```

Comment:
if you added this because the source file is `.xz`, it's kind of futile, cfr. https://github.com/hpcugent/easybuild-framework/issues/1376

also, in that case, it should be a build dep